.github/workflows/ci-slang-test-container.yml

```
@@ -125,6 +131,7 @@ jobs:
             -expected-failure-list tests/expected-failure-linux-gpu.txt
 
   test-slang-rhi:
+    if: false
```

actionlint (1.7.11)

[error] constant expression "false" in condition. remove the if: section

(if-cond)